install_files/ansible-base/roles/app-test/tasks/download_verify_tbb.yml

```diff
@@ -0,0 +1,69 @@
+---
+- name: Create system account user
```

Comment:
This code was written with the use case of supporting Tor Browser and multiple human users and a system account. Here we'll only be running the tests as a single user, whether that's `travis` or `vagrant`, so to simplify things it's best if we don't create this system account and perform operations as it.